src/transformers/models/depth_anything/configuration_depth_anything.py

```diff
@@ -67,6 +67,11 @@ class DepthAnythingConfig(PretrainedConfig):
             The index of the features to use in the depth estimation head.
         head_hidden_size (`int`, *optional*, defaults to 32):
             The number of output channels in the second convolution of the depth estimation head.
+        depth_estimation (`str`, *optional*, defaults to `"relative"`):
+            The type of depth estimation to use. Can be one of `["relative", "metric"]`.
```

Comment:
Perhaps we can rename this to either
* `depth_estimation_type`, to indicate that it's a type
* or `final_activation_function``? Which can be either "relu" or "sigmoid"
cc @amyeroberts

Reply:
Thanks for the feedback @NielsRogge! 

I think it makes more sense to use `depth_estimation_type` than `final_activation_function` since using a "metric" depth estimation type encompasses more than just switching the final activation function by Sigmoid - it also involves scaling the output by `max_depth`. So, I believe using `depth_estimation_type` would be more accurate for this concept